google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/JsonToProtoMessage.java

```diff
@@ -416,7 +416,9 @@ private static void fillRepeatedField(
             }
           }
           if (!added) {
-            if (val instanceof JSONArray) {
+            if (val instanceof byte[]) {
+              protoMsg.addRepeatedField(fieldDescriptor, val);
+            } else if (val instanceof JSONArray) {
```

Comment:
[no action required in this PR] It's a bummer to still see `org.json` used as a dependency here. I assume we have an issue tracking a migration to Gson?

Reply:
Yeah, for sure -- b/216657461 is tracking this work.